packages/twenty-server/src/engine/twenty-app/twenty-app.ts

```diff
@@ -0,0 +1,68 @@
+import { Module, Provider, Type as NestType } from '@nestjs/common';
+import { MODULE_METADATA } from '@nestjs/common/constants';
+import { ModuleMetadata } from '@nestjs/common/interfaces';
+import { APP_FILTER, APP_GUARD, APP_INTERCEPTOR, APP_PIPE } from '@nestjs/core';
+
+import { TWENTY_APP_METADATA } from 'src/engine/twenty-app/twenty-app.metadata';
+import { pick } from 'src/utils/pick';
+
+export interface TwentyAppMetadata extends ModuleMetadata {
+  configuration?: string;
+  resolvers?: Array<any>;
+  standardObjects?: Array<any>;
+  listeners?: Array<any>;
+  jobs?: Array<any>;
```

Comment:
We should properly type this to avoid problems, at least using `ModuleMetadata['imports']`